custom/icds/translations/integrations/client.py

```diff
@@ -196,3 +196,39 @@ def get_source_lang(self):
         :return: source lang code on transifex
         """
         return self.project_details().json().get('source_language_code')
+
+    def move_resources(self, hq_lang_code, target_project, version=None, use_version_postfix=True):
+        """
+        ability to move resources from one project to another
+        :param hq_lang_code: lang code on hq
+        :param target_project: target project slug on transifex
+        :param version: version if needed on parent resource slugs
+        :param use_version_postfix: to use version postfix in new project
+        :return: responses per resource slug
+        """
+        responses = {}
+        for resource_slug in self.get_resource_slugs(version):
+            lang = self.transifex_lang_code(hq_lang_code)
+            url = "https://www.transifex.com/api/2/project/{}/resource/{}/translation/{}/?file".format(
+                self.project, resource_slug, lang
+            )
+            response = requests.get(url, auth=self._auth, stream=True)
```

Comment:
This method is currently not called anywhere but was used for some dev work and was included because its useful.
I think its fine if it raises an exception.

Reply:
also the other places where we request to transifex is either as a celery task or within a POST Http request to HQ from the browser for a quick check, so both should be fine, considering celery task would just retry/fail eventually and the request to HQ would just throw an exception but would probably succeed on retry.
Also, i don't expect any of these to be very long requests. The only slightly longer(where we submit a set of translations) would be under celery tasks, so the only issue i can see happening would be an intermittent timeout which should resolve on retry otherwise it would be a bug that would need to be addressed.